sdk/scheduler/src/test/java/com/mesosphere/sdk/offer/evaluate/NamedVIPEvaluationStageTest.java

```diff
@@ -45,4 +49,40 @@ public void testDiscoveryInfoPopulated() throws Exception {
         Assert.assertTrue(vipLabel.getKey().startsWith("VIP_"));
         Assert.assertEquals(vipLabel.getValue(), "test-vip:80");
     }
+
+    @Test
+    public void testVIPIsReused() throws InvalidRequirementException {
```

Comment:
As for multiple offers, each offer evaluation round starts with a fresh `PodInfoBuilder`, so no state is retained between rounds.